utils.py

```diff
@@ -332,10 +334,19 @@ def are_datetimes_close(later_datetime, earlier_datetime):
 
 
 def generate_random_string(length):
+    """Generates a random string of the specified length.
+
+    Args:
+      length(Int): Length of the string to be generated.
+
+    Returns:
+      Random string of specified string.
+    """
```

Comment:
No string is specified here. Please change it to -- **str. Random string of specified length.** Do not forget to write datatype in each case. See my previous comments. You can refer previous PRs for reference also - [#4280](https://github.com/oppia/oppia/pull/4280#issuecomment-354727512)

Reply:
Sorry for that typo :sweat_smile: